### PR TITLE
chore: configure Husky pre-commit lint + prettify

### PR DESCRIPTION
**Pull request by colbr**
This should (hopefully) improve DX by making linting automatic.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,6 +20,11 @@
     "test": "jest",
     "test:watch": "jest --watch"
   },
+  "husky": {
+    "hooks": {
+      "pre-commit": "yarn lint"
+    }
+  },
```
Comment (joonchoi0918) [inline package.json:27]: Great!

Comment (joonchoi0918) [reply package.json:27]: nit: Can we add `prettify` command as well? So prettier first and lint sencond.
```
"husky": {
    "hooks": {
      "pre-commit": ["yarn prettify", "yarn lint"]
    }
  },
```

**Review by joonchoi0918 — APPROVED** [review]
Nice work! Thanks 🙏

File: package.json (hunk 1 above)
Comment (colbr) [reply package.json:27]: Sure thing - will add!

**Pull request merged**